### PR TITLE
API 500 Support for SAN Manager

### LINKED ISSUE #273: Support for SAN Manager - API500
### Scenario/Intent
Add support to SAN Manager on the API500.

### Environment Details
 - **oneview Cookbook:** Version: 2.3.0
 - **OneView Version:** 3.10
 - **chef-client Version:** 12.0+
 - **platform:** Any

### Steps to Reproduce
Try to use the resource SAN Manager on the API 500.

### Expected Result
This resource supports API 500.

### Actual Result
This resource does not support API 500.

### PR DESCRIPTION
**Pull request by aalexmonteiro**
### Description
Adding SAN Manager for HPE OneView API500 support.

### Issues Resolved
#273 

### Check List
- [] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
  - [] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [] New functionality has been documented in the README if applicable.
  - [] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [x] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## 2.3.1 (Unreleased)
 Adds API 500 support to the following HPE OneView resources:
   - oneview_scope
+  - oneview_san_manager
```
Comment (fgbulsoni) [inline CHANGELOG.md]: Should be in alphabetic order.

File: examples/san_manager.rb [hunk 2]
```diff
@@ -32,7 +32,7 @@
 end
 
 # Example: remove the SAN manager
-oneview_san_manager '172.18.15.1' do
+oneview_san_manager 'Remove 172.18.15.1' do
```
Comment (fgbulsoni) [inline examples/san_manager.rb]: This means that the resource is named `Remove 172.18.15.1`, which makes no sense. This modification *looks* untested.

**Review by fgbulsoni — CHANGES_REQUESTED** [review]
Change to changelog and example

File: examples/san_manager.rb (hunk 2 above)
Comment (aalexmonteiro) [reply examples/san_manager.rb]: This has been tested. I've probably accidentally deleted the data ```(name: '172.18.15.1')```.

Comment (fgbulsoni) [reply examples/san_manager.rb]: `(name: '172.18.15.1')` should not be necessary as long as the `title` is correct, a.k.a. Drop these modifications and it *should* work

File: examples/san_manager.rb [hunk 3]
```diff
@@ -34,5 +34,6 @@
 # Example: remove the SAN manager
 oneview_san_manager 'Remove 172.18.15.1' do
   client my_client
+  data(name: '172.18.15.1')
```
Comment (fgbulsoni) [inline examples/san_manager.rb]: Definitely should be dropped.

**Review by fgbulsoni — CHANGES_REQUESTED** [review]

**Review by fgbulsoni — APPROVED** [review]

**Pull request merged**